.github/stale.yml

```diff
@@ -0,0 +1,40 @@
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 30
+
+# Issues with these labels will never be considered stale. Set to `[]` to disable
+# We don't close any issue that is an enhancement or confirmed bug, but issues
+# waiting for reproduction cases and questions tend to get outdated.
+exemptLabels:
+  - "enhancement"
+  - "bug"
+  - "thinking"
+  - "docs"
+
+# Label to use when marking as stale
+staleLabel: "waiting-reply"
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: |
+  Hey there,
+  We wanted to check in on this request since it has been inactive for at least 60 days.
+  If you think this is still an important issue in the latest version of [Consul](https://github.com/hashicorp/consul/compare/)
```

Comment:
The URL for the hyperlink seems wrong here.

Reply:
What do you mean? 
I meant it to be the compare so they can compare current branches vs master, but maybe that's not a good link to throw there?

Reply:
That link opens a comparison of master <-> master which will always be empty. Maybe a better link is to the changelog?

Reply:
^^ i really like this idea, let me update it :)